content/learn/book/welcome/plugins/_index.md

```diff
@@ -1,16 +1,150 @@
+        group.add(bevy_app::ScheduleRunnerPlugin::default());
+    }
+}
+```
+
+The `CorePlugin` handles low-level fundamentals such as updating app time, while the `ScheduleRunnerPlugin` sets up the main game loop to run repeatedly over time.
+
 ## `DefaultPlugins`
 
+Bevy's `DefaultPlugins` are intended to get up and running with a "standard game" as quickly as possible. Let's take a look at the source again:
+
+```rust
+pub struct DefaultPlugins;
+
+impl PluginGroup for DefaultPlugins {
+    fn build(&mut self, group: &mut PluginGroupBuilder) {
+        group.add(bevy_log::LogPlugin::default());
+        group.add(bevy_core::CorePlugin::default());
+        group.add(bevy_transform::TransformPlugin::default());
+        group.add(bevy_diagnostic::DiagnosticsPlugin::default());
+        group.add(bevy_input::InputPlugin::default());
+        group.add(bevy_window::WindowPlugin::default());
+        group.add(bevy_asset::AssetPlugin::default());
+        group.add(bevy_scene::ScenePlugin::default());
+
+        #[cfg(feature = "bevy_render")]
+        group.add(bevy_render::RenderPlugin::default());
+
+        #[cfg(feature = "bevy_sprite")]
+        group.add(bevy_sprite::SpritePlugin::default());
+
+        #[cfg(feature = "bevy_pbr")]
+        group.add(bevy_pbr::PbrPlugin::default());
+
+        #[cfg(feature = "bevy_ui")]
+        group.add(bevy_ui::UiPlugin::default());
+
+        #[cfg(feature = "bevy_text")]
+        group.add(bevy_text::TextPlugin::default());
+
+        #[cfg(feature = "bevy_audio")]
+        group.add(bevy_audio::AudioPlugin::default());
+
+        #[cfg(feature = "bevy_gilrs")]
+        group.add(bevy_gilrs::GilrsPlugin::default());
+
+        #[cfg(feature = "bevy_gltf")]
+        group.add(bevy_gltf::GltfPlugin::default());
+
+        #[cfg(feature = "bevy_winit")]
+        group.add(bevy_winit::WinitPlugin::default());
+
+        #[cfg(feature = "bevy_wgpu")]
+        group.add(bevy_wgpu::WgpuPlugin::default());
+    }
+}
+```
+
+As you can see, there are some added niceties (logging and diagnostics, scenes, input and windowing) as well as a number of plugins controlled by feature flags.
+These are heavier dependencies that you may want to disable for some games (or platforms), and replace with your own alternatives.
+To do so, we can use [cargo features](https://doc.rust-lang.org/cargo/reference/features.html) to disable them in our game.
+
+In your `Cargo.toml` file, disable default features and opt-in to the [features you want](https://github.com/bevyengine/bevy/blob/main/docs/cargo_features.md):
+
+```toml
+[dependencies]
+bevy = {"0.5", , default-features = false, features = ["..."]}
+```
+
+As shown in the [plugin_group.rs](https://github.com/bevyengine/bevy/blob/latest/examples/app/plugin_group.rs) example, you can also configure plugin groups from within Bevy itself.
+
 ## Writing your own plugins
 
+Plugins are simply collections of code that modify the `AppBuilder`.
```

Comment:
This refers to the `AppBuilder`, but at this point of the book relevant talk about the app has not been done yet.

Reply:
Provided some basic context here :)